THEME.md

```diff
@@ -205,25 +205,46 @@ It's a good idea to add this to the top of the template to improve the styling o
 
 ## Action Injection
 
-The following will inject the action link (or button) into the e-mail:
+The following will inject the action link (or button) into the e-mail.
 
 ```html
 <!-- Action -->
+<% if(locals.action.constructor !== Array){ %>
```

Comment:
Please add a space after `if` and before `(` as per the project's code style. Please also do this in all other places in your PR (in theme files).